eventqueue/eventqueue_test.go

```diff
@@ -107,13 +107,13 @@ func TestPullBlocksUntilAvailable(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Millisecond)
 	defer cancel()
 
-	got, err := q.Pull(ctx)
+	_, err := q.Pull(ctx)
```

Comment:
Go usually prefers limited scopes for variables. This is especially true about the err variable which gets used a lot. Limiting its scope prevents programming errors where we might forget to check it.

Now that we don't use the return value, we can limit the scope of err to the if statement:
if _, err := q.Pull(ctx); err != nil {
    t.Fatal("Pull => expected timeout error, got nil")
}

Reply:
Alright, I fixed that. Please note that I adjusted your code  to match err == nil, as the original if statement did exactly that, and the tests would fail otherwise.

Thanks for the advice on in-if-statement checking!

Reply:
Of course, this was a case of "tired eyes" on my end. Thank you.